### PR TITLE
Make username field read-only

### PR DESCRIPTION
**Pull request by bhavenst**
Not sure if there is a lighter weight way to do this, but I didn't find one.  This makes the username field read-only and removes the cursor/selectability..

### CODE REVIEW

**Comment by bhavenst** [comment]
image

image

File: ui/src/app/Login/FixedUsernameLoginForm.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
+import * as React from 'react';
```
Comment (tznamena) [inline ui/src/app/Login/FixedUsernameLoginForm.tsx:1]: Is this re-implementing the whole login form?

**Comment by tznamena** [comment]
Let's split this up into two PRs:
- update of the doc link
- making user name field disabled in log-in form

File: ui/src/app/Login/FixedUsernameLoginForm.tsx (hunk 1 above)
Comment (bhavenst) [reply ui/src/app/Login/FixedUsernameLoginForm.tsx:1]: Yes it is.  I have very rudimentary react knowledge, so perhaps there is a better way.  I couldn't find any way to modify the encapsulated component (loginform) from outside, so did it by copying/modifying it.  Seems like you can inherit/extend, but again I didn't see how to modify already included components by doing that.  Open to any suggestion.

**Comment by bhavenst** [comment]
OK, they are split.

File: ui/src/app/Login/FixedUsernameLoginForm.tsx (hunk 1 above)
Comment (qjones7) [reply ui/src/app/Login/FixedUsernameLoginForm.tsx:1]: thats the only way to do it. I tired many ways but you would have to recreate the form.

Comment (bhavenst) [reply ui/src/app/Login/FixedUsernameLoginForm.tsx:1]: I figured that you tried to do that as well, thanks.

**Comment by bhavenst** [comment]
Additional update requested by Qi.  The password field and helper text now becomes red/has icon w/ wrong password.  
image

**Comment by tznamena** [comment]
This is a bit of a scope creep, let's not add any more features otherwise we will never merge this.

**Review by tznamena — APPROVED** [review]

**Review by benjdudas — APPROVED** [review]

**Pull request merged**